quality/postman/README.md

```diff
@@ -0,0 +1,46 @@
+EmoDB http endpoints tests in Postman
+======================================
+
+Written by [Bazaarvoice](http://www.bazaarvoice.com)
+
+Introduction
+------------
+
+Postman tests are tests written in [Postman](https://www.postman.com/) for testing different business functions
+via http endpoints exposed by EmoDB application.
+
+Below can be found two ways of running Postman tests even though all ways provided by Postman are supported. 
+
+Running tests via Postman GUI interface
+---------------------------------------
+Postman keeps its documents always up to date and provide manual with graphical representation of all screens involved.
+Manual for running postman tests via gui can be found on [Postman webpage](https://learning.postman.com/docs/running-collections/intro-to-collection-runs/)
+
+Running tests via command line 
+------------------------------
+1. Install command line tool which allows running postman tests via command line:
+
+```
+npm install -g newman
+```
+
+2.Execute preconditions postman collection which setups api keys in case they are missing on server used for testing:
+
+```
+newman run "tests_preconditions.postman_collection.json" -e "sor-1-_table-table.postman_environment.json" --env-var "api_key=$EMODB_POSTMAN_CERT_ADMIN_APIKEY" --env-var "baseurl_dc1=$Region1 --env-var "baseurl_dc2=$Region2 --env-var "api_key_no_rights=$EMODB_POSTMAN_CERT_APIKEY_SOR_READ"
```

Comment:
add example how to run newman locally with all variables set explicitly

Reply:
Variables names are renamed to bring more clarity. 
Example of running on local has been added; however, I left example with abstract variables names due to impossibility to use all possible/ needed variables from other servers on localhost (e.g. facade tests, on local there is not sor read api key expected to be present so no variable can supplied for this as well)